src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -107,15 +110,42 @@ class ConfigRepo:
     Wraps accessing ~/.orquestra/config.json
     """
 
-    def list_config_names(self) -> t.Sequence[ConfigName]:
-        config_entries = sdk.RuntimeConfig.list_configs()
-        return [
-            # .list_configs() should probably already return "ray" and "in_process".
-            # TODO: fix that in https://zapatacomputing.atlassian.net/browse/ORQSDK-674
-            _config.RAY_CONFIG_NAME_ALIAS,
-            _config.IN_PROCESS_CONFIG_NAME,
-            *config_entries,
-        ]
+    @staticmethod
+    def list_config_names() -> t.Sequence[ConfigName]:
+        return sdk.RuntimeConfig.list_configs()
+
+    @staticmethod
+    def store_token_in_config(uri, token):
+        runtime_name = RuntimeName.QE_REMOTE
+        config_name = _config.generate_config_name(runtime_name, uri)
+
+        config = sdk.RuntimeConfig(
+            runtime_name,
+            name=config_name,
+            bypass_factory_methods=True,
+        )
+        setattr(config, "uri", uri)
+        setattr(config, "token", token)
+        _config.save_or_update(config_name, runtime_name, config._get_runtime_options())
+
+        return config_name
+
+
+class QeClientRepo:
```

Comment:
**suggestion**: recently @jamesclark-Zapata convinced me to go with all caps when using acronyms inside camel case names
```suggestion
class QEClientRepo:
```